libs/components/lists/src/lib/modules/sort/sort-item.component.ts

```diff
@@ -30,63 +30,66 @@ export class SkySortItemComponent implements OnInit, OnChanges, OnDestroy {
    * Indicates whether the sorting option is active.
    */
   @Input()
-  public active: boolean;
+  public active: boolean | undefined;
 
   /**
    * Fires when a sort item is selected.
    */
   @Output()
-  public itemSelect: EventEmitter<any> = new EventEmitter();
+  public itemSelect: EventEmitter<any> = new EventEmitter(); // TODO: EventEmitter is of type <void>
```

Comment:
Let's put the keywords "breaking change" somewhere in the TODO so they're easy to find later.
```
// TODO: Change to `EventEmitter<void>` in a breaking change.
```

Reply:
fixed in: [respond to comment](https://github.com/blackbaud/skyux/pull/686/commits/79dceb37a5310453c7a1ceee9ce8171431f6d3e2)